src/core/font_renderer.js

```diff
@@ -25,7 +25,9 @@ import { getGlyphsUnicode } from "./glyphlist.js";
 import { StandardEncoding } from "./encodings.js";
 import { Stream } from "./stream.js";
 
-function getLong(data, offset) {
+// TODO: use DataView and its methods.
+
+function getUint32(data, offset) {
```

Comment:
Is this name completely accurate, i.e. shouldn't it be `getInt32` instead?

Please note that we have essentially identical code on the `Stream`-instances, but there we use the `getInt32` name:
https://github.com/mozilla/pdf.js/blob/749a3fa6994a656f2c9427cd932945168c74b943/src/core/base_stream.js#L70-L76
Furthermore, we have a helper function named `readUint32` which actually ensures that the value is in fact unsigned:
https://github.com/mozilla/pdf.js/blob/749a3fa6994a656f2c9427cd932945168c74b943/src/core/core_utils.js#L237-L245

Reply:
Interesting...
You're right, it's really getInt32 here but in reading for example cmap specs:
https://docs.microsoft.com/en-us/typography/opentype/spec/cmap
and in particular:
https://docs.microsoft.com/en-us/typography/opentype/spec/cmap#encoding-records-and-encodings
we should really have an uint32 in some places (maybe all).
I suppose that in general it doesn't hurt to get an int32 because we don't have font files with several Gb but it's a bug.

Reply:
I updated the code to really have an uint32 instead of an int32.
I checked in parallel the specs to be sure that an uint32 is expected.